crypto/engine/eng_cryptodev.c

```diff
@@ -1082,13 +1082,21 @@ cryptodev_asym(struct crypt_kop *kop, int rlen, BIGNUM *r, int slen,
         return (ret);
 
     if (r) {
-        kop->crk_param[kop->crk_iparams].crp_p = calloc(rlen, sizeof(char));
+        kop->crk_param[kop->crk_iparams].crp_p = OPENSSL_zalloc(rlen * sizeof(char));
+        if (kop->crk_param[kop->crk_iparams].crp_p == NULL)
+            return (ret);
         kop->crk_param[kop->crk_iparams].crp_nbits = rlen * 8;
         kop->crk_oparams++;
     }
     if (s) {
         kop->crk_param[kop->crk_iparams + 1].crp_p =
-            calloc(slen, sizeof(char));
+            OPENSSL_zalloc(slen * sizeof(char));
```

Comment:
```sizeof(char)``` is defined to be one.  please fix that, squash them into a single commit.  i'll then plus-one.